specification/batch/Azure.Batch/client.tsp

```diff
@@ -4,19 +4,26 @@ import "./main.tsp";
 using Azure.ClientGenerator.Core;
 
 // operation overrides
-@@access(Azure.Batch.BatchOps.ListOperation, Access.internal, "java");
-@@access(Azure.Batch.BatchOps.ReadOperation, Access.internal, "java");
-@@access(Azure.Batch.BatchOps.DeleteOperation, Access.internal, "java");
-@@access(Azure.Batch.BatchOps.CreateOperation, Access.internal, "java");
-@@access(Azure.Batch.BatchOps.UpdateOperation, Access.internal, "java");
-@@access(Azure.Batch.BatchOps.ReplaceOperation, Access.internal, "java");
-@@access(Azure.Batch.BatchOps.PostOperation, Access.internal, "java");
-@@access(Azure.Batch.BatchOps.ResourceInfoOperation, Access.internal, "java");
-@@access(Azure.Batch.BatchOps.ResourceInfoOperationWithAdditionalResponse,
+@@access(Azure.Batch.ListOperation, Access.internal, "java");
+@@access(Azure.Batch.ReadOperation, Access.internal, "java");
+@@access(Azure.Batch.DeleteOperation, Access.internal, "java");
+@@access(Azure.Batch.CreateOperation, Access.internal, "java");
+@@access(Azure.Batch.UpdateOperation, Access.internal, "java");
+@@access(Azure.Batch.ReplaceOperation, Access.internal, "java");
+@@access(Azure.Batch.PostOperation, Access.internal, "java");
+@@access(Azure.Batch.FileOperation, Access.internal, "java");
+@@access(Azure.Batch.HeadFileOperation, Access.internal, "java");
+@@access(Azure.Batch.ResourceInfoOperationWithAdditionalResponse,
   Access.internal,
   "java"
 );
 
+// Api overrides
+@@access(Azure.Batch.Batch.poolExists, Access.internal, "csharp");
```

Comment:
`Azure.Batch.Batch` looks strange - can we rename one of these?

Reply:
Maybe just rename the `Batch` interface to `BatchClient` so this is `@@access(Azure.Batch.BatchClient)`? This seems to match the pattern I see other teams using.

Reply:
changed it to BatchClient but it did have the side effect of changing the operationIds.